res/controllers/Denon-HC1000S-scripts.js

```diff
@@ -0,0 +1,920 @@
+////////////////////////////////////////////////////////////////////////
+// Controller: Denon HC1000S
+////////////////////////////////////////////////////////////////////////
+var DenonHC1000S = {};
+
+////////////////////////////////////////////////////////////////////////
+
+// SERATO-Certification ->
+//The SysEx message to send to the controller to force the midi controller
+// to send the status of every item on the control surface.
+var ControllerStatusSysex = [0xF0, 0x00, 0x20, 0x7F, 0x03, 0x01, 0xF7];
+
+
+/*
+DN-HC1000S todo notes:
+	4 Deck support (with C+D)
+
+optional (local):
+	DVS control w/TraktorSoundCard Ctrl (THRU)
+------
+	[done] LED for Sampler/Play/Cue if 2nd Shift are pressed
+	[done] DVS control software (place these functions somewhere)
+	[done] 	Play & Cue: 	use LoadSample button as 2nd Shift with Cue1/2 buttons
+	[done] RecordButton (shift TrackSelect)
+	[done]SamplerLEDs: overwrite HotCue Lights with SamplerLoaded-Status if "load" buttons are pressed
+	[done]AutoLoop: make via script to handle LED (off if exiting via Loop);
+	[done]LEDs: explore binding hotcues and loop buttons so the lights react on the actual actual status (e.g. if changed via screen UI)
+	[done]LoopLED: reset on new track
+	[done]LED blinking if r/l-shift pressed and sample loaded
+	[done] LoopCut Wheel: add secondary dial (if pressed) to move loop right/left
+	[done] Samples: use LoadSample buttons as 2nd Shift for Cue3/4/5
+	[done] Tap
+	[done] Fwd/Back button
+	[done] Instant Double (shift load track support)
+*/
+/////INIT///////
+
+DenonHC1000S.init = function(id, debug) {
+    DenonHC1000S.id = id;
+    DenonHC1000S.debug = debug;
+    // create an instance of your custom Deck object for each side of your controller
+    DenonHC1000S.leftDeck = new DenonHC1000S.Deck([1, 3], 1);
+    DenonHC1000S.rightDeck = new DenonHC1000S.Deck([2, 4], 2);
+
+    engine.setValue("[Master]", "num_decks", 4);
+    var leds = [0x11, 0x13, 0x15, 0x17, 0x19, /* cues deckA*/
+        0x24, 0x40, 0x2B, /* loops deckA*/
+        //0x27,0x26,/* play,cue */
+        //0x08,0x09,/* key lock, sync*/
+        0x09, , /* sync deckA*/
```

Comment:
Here is one of the errors:

```suggestion
        0x09, /* sync deckA*/
```